eng/sdk_files/Emscripten.Python.props

```diff
@@ -1,5 +1,13 @@
 <Project>
   <PropertyGroup>
     <EmscriptenPythonToolsPath>$(MSBuildThisFileDirectory)..\tools\</EmscriptenPythonToolsPath>
+
+    <EmscriptenPythonBinPath Condition="!$([MSBuild]::IsOSPlatform('WINDOWS'))">$(EmscriptenPythonToolsPath)bin\</EmscriptenPythonBinPath>
```

Comment:
Just to be super sure - it's using a back-slash as a path separator on non-Windows OSes. I know MSBuild does some magic around this, but presumably this works for your scenario too?

Reply:
Yeah, that'll be fine